examples/BasicConsoleAppILogger/Program.cs

```diff
@@ -31,25 +41,28 @@ static void Main(string[] args)
                 // Application Insights for all categories.
                 builder.AddFilter<Microsoft.Extensions.Logging.ApplicationInsights.ApplicationInsightsLoggerProvider>("", LogLevel.Trace);
                 builder.AddApplicationInsights("---Your AI instrumentation key---");
-            });
 
-            // Add application insights for Kubernetes.
-            services.AddApplicationInsightsKubernetesEnricher();
+                // Optional: Show the logs in console at the same time
+                builder.AddConsole();
```

Comment:
This call requires the `Microsoft.Extensions.Logging.Console` package, which is not referenced by the sample

Reply:
@GeorgeTsiokos Thanks for pointing this out. Please file an issue and I'll fix it.